charts/external-dns/templates/deployment.yaml

```
@@ -132,6 +134,55 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+        {{- if eq $providerName "webhook" }}
+        {{- with .Values.provider.webhook }}
+        - name: webhook
+          {{- with .image }}
+          {{- if or (empty .repository) (empty .version) }}
+            {{- fail "ERROR: webhook provider needs an image repository and a version" }}
+          {{- end }}
+          image: {{ .repository }}:{{- .version }}
```

```suggestion
        - name: webhook
          {{- with .image }}
          {{- if or (empty .repository) (empty .tag) }}
            {{- fail "ERROR: webhook provider needs an image repository and a tag" }}
          {{- end }}
          image: {{ include "external-dns.webhookImage" . }}
```
If you add the `external-dns.webhookImage` function to match the current image function. You could also move the validation into there to keep this template clean.